coopterrones/rancid-tomatoes
# Feature/iteration 3

# Pull Request


#### What's this PR do?
1. Adds ApiCalls to separate file
2. Add chosen video requests 
3. Add youtube trailer to chosen movies display 
4. Add iFrame positioning 
5. Styling changes
#### Where should the reviewer start?
* Line 47 `App.js`
* Line 19 `apiCalls.js`
* Line 37 `chosenMovie.js`
* Line 9 `chosenMovie.scss`

#### How should this be manually tested?
* Test that all functionality is working using `npm start` 
* check for errors in console 


#### Any background context you want to provide?
* Starting to implement testing next! Lets goooo
* Look at our pretty f'n trailer! So dooppee .. Great work. 
#### Screenshots (if appropriate):
![Screen Shot 2020-12-03 at 4 05 14 PM](https://user-images.githubusercontent.com/67164959/101099127-623ab500-3581-11eb-8a01-e9b7e0d79bdf.png)
